checkstyle-tester/diff.groovy

```diff
@@ -15,6 +15,14 @@ static void main(String[] args) {
             baseBranch = 'master'
         }
 
+        def baseConfig = cliOptions.baseConfig
+        def patchConfig = cliOptions.patchConfig
+        def config = cliOptions.config
+        if (config) {
```

Comment:
@rnveach 
This looks weird but Groovy treats config as false if ```--config``` is not set and as true which may be cast to a string when ```--config``` is set)